src/ResourceManagement/Insights/Microsoft.Azure.Insights/Generated/Management/Insights/Models/LogProfileCreateOrUpdateParameters.cs

```diff
@@ -1,90 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
LogProfileCreateOrUpdateParameters has been replaced by LogProfileResource which contains all the fields required by the PUT operation for LogProfiles. Once serialized, the two objects look almost identical.
Aman suggested the change since it breaks nothing and removes one model from the spec. Certainly I had to change the PowerShell code a little bit, but the customers won't see any change.